blackbox/docs/best_practice/rolling_upgrade.txt

```diff
@@ -268,5 +269,5 @@ Step 4: Repeat
 
 Repeat step two, three, and four for all other nodes.
 
-.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
 .. _accompanying notes: https://crate.io/a/upgrade-your-cluster-to-0-55/
+.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
```

Comment:
Not referenced in this text, I think we should also include a note that backing up date before the rolling upgrade is highly advised...